ambry-api/src/main/java/com.github.ambry/config/ReplicationConfig.java

```diff
@@ -181,5 +189,6 @@ public ReplicationConfig(VerifiableProperties verifiableProperties) {
     replicaMetadataRequestVersion =
         verifiableProperties.getShortInRange("replication.metadata.request.version", (short) 1, (short) 1, (short) 2);
     replicationEnabledWithVcrCluster = verifiableProperties.getBoolean("replication.enabled.with.vcr.cluster", false);
+    vcrRecoveryPartitions = verifiableProperties.getString("vcr.recovery.partitions", "");
```

Comment:
+1 to Yingyi's suggestion about doing the string splitting here and making the config a `Set<String>`

Reply:
see reply below for CloudToReplicationManager.